server/utils/get_request_data_csv.py

```diff
@@ -27,19 +27,22 @@ def get_311_request_data(start_date, end_date):
         skip += REQUESTS_BATCH_SIZE
         if len(data) < skip:
             break   
-    data_final = pd.DataFrame(all_requests) 
+    data_final = pandas.DataFrame(all_requests) 
     data_final.sort_values(by='createdDate', inplace = True, ignore_index = True)
-    data_final.to_csv('data_final.csv')
     return data_final
 
 def main():
+    """Prints out the preview of the dataframe data_final in the command line. The result is written to a csv file and saved in the current working directory of the user.

```

Comment:
Nit: this line is really long; generally, we try to stick to 80 chars per line. Could you put the second sentence on the next line?